packages/ens-controller/src/EnsController.ts

```diff
@@ -193,6 +260,74 @@ export class EnsController extends BaseControllerV2<
     });
     return true;
   }
+
+  #getNetworkEnsSupport(network: NetworksChainId) {
+    return Boolean(ensNetworkMap[network]);
+  }
+
+  lookup(ensName: string | Promise<string>): Promise<string | null> {
```

Comment:
These two methods (`lookup` and `reverse`) have odd signatures; why are we accepting a Promise? It seems that we always pass in a string, never `Promise<string>`.

Also, can these be internal functions? They don't appear to be a part of the public API on the extension.